modules/apps/27-interchain-accounts/keeper/account_test.go

```diff
@@ -67,3 +68,74 @@ func (suite *KeeperTestSuite) TestInitInterchainAccount() {
 		})
 	}
 }
+
+func (suite *KeeperTestSuite) TestInitChannel() {
```

Comment:
this should test that when a channel is closed and the portID is set to inactive, a new channel can be opened